app/javascript/src/Policies/components/PolicyConfig.jsx

```diff
@@ -68,16 +69,17 @@ const PolicyConfig = ({policy, actions}: Props) => {
           formData={data}
           onSubmit={onSubmit(policy)}
         >
-          <button className="btn btn-info" type="submit">Update Policy</button>
+          <Button className="btn-info" type="submit">Update Policy</Button>
         </Form>
       }
       { removable &&
-        <div
-          className="PolicyConfiguration-remove btn btn-danger btn-sm"
+        <Button
+          className="PolicyConfiguration-remove"
```

Comment:
We should start removing the old styles, since they are overridden by PF4. In this case:
```css
    &-remove {
      background-color: white;
      border-color: $dontColor;
      color: $dontColor;
    }
```
```suggestion
```